var/plugins/barmanager.py

```diff
@@ -333,6 +334,50 @@ def sendCurrentVote():
         spads.slog("Unhandled exception: " + str(sys.exc_info()
                    [0]) + "\n" + str(traceback.format_exc()), 0)
 
+
+def getBarGameType(teamsize, teamcount):
+    spadsConf = spads.getSpadsConf()
```

Comment:
If the majority of calls to getBarGameType have parameters, then there is no need to fetch getSpadsConf. Also, you can use default args of = None to this function.

Reply:
If getSpadsConf is an expensive operation does it help if I use ChobbyState variable instead? I have added a new commit.